tests/python/test_files/pylint_test.py

```diff
@@ -1,5 +1,4 @@
 import sys
-import platform  # Unused import
```

Comment:
I think everything in ` tests/python/test_files/` shouldnt be modified - they are intentionally bad syntax